### ISSUE TITLE
adult dataset with id 179 is wrong 

### ISSUE THREAD

**Issue by benjaminschober** [openml/OpenML#189]
hey there. Im working with bernd for my bachelor thesis and i saw a problem with the adult dataset id 179. the feature age shouldn't be a nominal scaled variable in the range of 0 to 4. at the uci repository, where i think it is from, it is a continous variable > 16.

**Comment by berndbischl** [comment]
Also 
a) there is no source ref to UCI
b) and there should be at least a short description of the data
(or for now at least a link to UCI, so I can read up on details there)

**Comment by joaquinvanschoren** [comment]
Hmm, there are at least two datasets:
http://openml.org/d/16  which has some discretized features
http://openml.org/d/1119  which is the same as the UCI original training set, but does not include the UCI test data

I would propose to rename the second one as another version of the first one, and then add a new version which has the original data with train and test sets merged?

The description can be easily added online (if you log in an edit button appears).

**Comment by berndbischl** [comment]
I am getting confused now?
So we have 3 versions? 179, 16 and 1119?

Ok, for later we can edit / add the description. Can we also add a src ref?

What happens if a another user later deletes a description? Is that possible too?

**Comment by berndbischl** [comment]
Actually 16 seems to be something else I guess.

**Comment by joaquinvanschoren** [comment]
Sorry, that should have been #179. My mistake. I'll upload a new version of the adult dataset tomorrow.

Yes, you can easily add a source ref in the wiki, I just did it :)
http://openml.org/d/179

Other users can change the description, but a history is kept, so it is easy to revert an edit, and if necessary block vandals.

**Comment by berndbischl** [comment]
1) You linked Iris on UCI in the Source Link. I corrected that. 

2) Can I see the history of commit messages on the webpage? That would be cool to see the applied changes of the past? Issue request?

3) New original adult data: This would be cool thanks!

4) Regarding that version in 179: 
Where is this from? Who discretized this? Can we try to say this at least in the description?

**Comment by joaquinvanschoren** [comment]
1) Haha, I noticed the same, we must have done it simultaneously
2) Already done. When you click edit, an 'history' button shows up as well
3) Ok, will do that tomorrow. Too sleepy now :)
4) Jan added this one, not sure where it came from.

**Comment by berndbischl** [comment]
4) Lets try to avoid that in the future. Maybe add some warning / text in the mask where you upload data?
This thing also relates to "being reproducible".....

**Comment by joaquinvanschoren** [comment]
Hi, I added the new dataset: http://openml.org/d/1590
And created a classification task: http://openml.org/t/7592

4) This has to do with data provenance, which is indeed important. I can
add a line asking to explain where the data comes from and how it was
altered.

Cheers,
joaquin

**Comment by berndbischl** [comment]
4) Do it and close?

**Comment by joaquinvanschoren** [comment]
Done in new website. Closing.
